### DISCUSSION TITLE
8088 Domination in 86Box?

### DISCUSSION THREAD

**Discussion by jadasse** [86Box/86Box#2337] [Q&A] [answered]
How do I get the files for 8088 Domination into my IBM XT VM? I have tried a lot, but I am losing hope.

**Comment by telanus** [comment, marked as answer]
Easiest way would beusing winimage. Just create a backup of the virtual harddrive.

Just open the harddrive in winimage. Create a folder, and inject the files, either by using the menu or dragging and dropping the files.

**Reply by jadasse** [reply]
I'm going to try it

**Reply by jadasse** [reply]
It works! Also, I used XTIDE to mount the hdd for it to work. I can post the VM if anyone wants
